CHANGELOG.md

```diff
@@ -5,12 +5,22 @@ Notable changes to this project are documented in this file. The format is based
 ## [Unreleased]
 
 Breaking changes:
+- Updated code for PureScript 0.14 (#120)
```

Comment:
I think it's clearer to phrase this as "Added support for PureScript 0.14, and dropped support for all previous versions"